docs/operators/windows_eventlog_input.md

```diff
@@ -9,10 +9,10 @@ The `windows_eventlog_input` operator reads logs from the windows event log API.
 | `id`            | `windows_eventlog_input` | A unique identifier for the operator                                                                                           |
 | `output`        | Next in pipeline         | The connected operator(s) that will receive all outbound entries                                                               |
 | `channel`       | required                 | The windows event log channel to monitor                                                                                       |
-| `max_reads`     | 100                      | The maximum number of records read into memory, before beginning a new batch                                                   |
+| `max_reads`     | 100                      | The maximum number of bodies read into memory, before beginning a new batch                                                   |
 | `start_at`      | `end`                    | On first startup, where to start reading logs from the API. Options are `beginning` or `end`                                   |
-| `poll_interval` | 1s                       | The interval at which the channel is checked for new log entries. This check begins again after all new records have been read |
-| `write_to`      | $                        | The record [field](/docs/types/field.md) written to when creating a new log entry                                              |
+| `poll_interval` | 1s                       | The interval at which the channel is checked for new log entries. This check begins again after all new body have been read |
```

Comment:
> This looks great. A few very minor suggestions.
> 
> It looks like there are a few more files to update.
> 
> ```
> > grep -Iris record .
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/types/severity.md:  "record": {
> ./docs/types/severity.md:  "record": {}
> ./docs/operators/windows_eventlog_input.md:  "record": {
> ./docs/operators/k8s_event_input.md:  "record": {
> ./docs/operators/restructure.md:`value_expr` is an [expression](/docs/types/expression.md) with access to the `record` object
> ./docs/operators/restructure.md:<tr><td> Input record </td> <td> Output record </td></tr>
> ./docs/operators/restructure.md:The `remove` op removes a field from a record.
> ./docs/operators/restructure.md:<tr><td> Input record </td> <td> Output record </td></tr>
> ./docs/operators/restructure.md:<tr><td> Input record </td> <td> Output record </td></tr>
> ./docs/operators/restructure.md:<tr><td> Input record </td> <td> Output record </td></tr>
> ./docs/operators/restructure.md:<tr><td> Input record </td> <td> Output record </td></tr>
> ```
> 
> I omitted these lines from the above output, because I believe they use record in a different context and should be left as is:
> 
> ```
> ./docs/operators/windows_eventlog_input.md:             "record_id": 1,
> ./operator/builtin/input/windows/subscription.go:               return EvtSubscribeStartAtOldestRecord
> ./operator/builtin/input/windows/xml.go:        RecordID    uint64      `xml:"System>EventRecordID"`
> ./operator/builtin/input/windows/xml.go:                "record_id":   e.RecordID,
> ./operator/builtin/input/windows/api.go:        // EvtSubscribeStartAtOldestRecord is a flag that will subscribe to all existing and future events.
> ./operator/builtin/input/windows/api.go:        EvtSubscribeStartAtOldestRecord uint32 = 2
> ./operator/builtin/input/windows/xml_test.go:           RecordID: 1,
> ./operator/builtin/input/windows/xml_test.go:           "record_id":   uint64(1),
> ```

thanks a lot!